src/components/Nav.tsx

```diff
@@ -15,27 +14,32 @@ const StyledNav = styled.div`
   z-index: 10;
 `;
 
-const StyledH1 = styled.h1`
+const StyledLink = styled.a.attrs(props => {
+  href: props.href
+})`
```

Comment:
Why do we not want to use the gatsby link component when we use gatsby for this website?

If you look at the preview deploy in the pull request you can see how much slower it is at loading the different pages under the nav, compared to the previous solution.

This might be a valid solution to a problem, but I don't know enough about the gatsby Link component to tell if this is a upgrade or not. Calling @sondr3. But I feel like this is a bad way to do it when we use gatsby to handle history, lazy loading etc.